### PR TITLE
Include git commit instruction

### PR DESCRIPTION
**Pull request by aayushmau5**
This PR:
- adds git commit section in each lesson
- creates a mix task to run the script: `mix add_git_section`

Closes https://github.com/DockYard-Academy/beta_curriculum/issues/153

### CODE REVIEW

**Comment by aayushmau5** [comment]
@BrooklinJazz created a WIP PR. I have also added the script file which I used to automate adding git section to files. I'll remove this when PR is ready to merge.

Please review :)

**Comment by BrooklinJazz** [comment]
@aayushmau5 This is looking great! Let me know when it's not WIP. I'll handle the merge conflicts and merge. (otherwise, you'll have to keep resolving them, so I'll just do it right before I approve and merge)

**Comment by aayushmau5** [comment]
@BrooklinJazz sure!

I'll remove the script file, then I think it will be ready to merge.

**Comment by BrooklinJazz** [comment]
Also @aayushmau5 I wonder if we can keep the script file?

We could add a quick check to see if the current file already contains the git message. Otherwise, run the file change as normal.
This way, we can use the script for new content. It's also a good example for future scripts.

Extra bonus points if you create a mix task to run the script!

**Comment by aayushmau5** [comment]
@BrooklinJazz alright. That makes sense. 

I'll do that :). Perhaps put these scripts into utils directory or create a scripts directory as well.

**Comment by BrooklinJazz** [comment]
A scripts directory or putting it into `utils` works for me! I don't have a strong preference

**Comment by BrooklinJazz** [comment]
Thank you for the awesome work @aayushmau5! I'll go through and handle conflicts then merge once I have some time! Signing off for tonight 😴

File: utils/lib/mix/tasks/add_git_section.ex [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  ]
+  @ignore_exercise_files [
+    "command_line_family_tree.livemd",
+    "github_collab.livemd",
+    "github_engineering_journal.livemd",
+    "livebook_recovery.livemd"
+  ]
+
+  defp get_paths do
+    base_path = Path.expand("../")
```
Comment (aayushmau5) [inline utils/lib/mix/tasks/add_git_section.ex:21]: @BrooklinJazz Couldn't think of a better approach here. 
This task should be executed from the `utils/` directory.

**Comment by aayushmau5** [comment]
@BrooklinJazz created mix task for the script, `mix add_git_section`. Please review :)

File: utils/lib/mix/tasks/add_git_section.ex (hunk 1 above)
Comment (BrooklinJazz) [reply utils/lib/mix/tasks/add_git_section.ex:21]: Seems like a reasonable expectation.

**Comment by BrooklinJazz** [comment]
Great Work!

**Pull request merged**